applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -72,27 +72,45 @@ See :ref:`nrf_desktop_memory_layout` for details.
 Configuring the B0 bootloader
 =============================
 
-To enable the B0 bootloader, select the :kconfig:option:`CONFIG_SECURE_BOOT` Kconfig option in the application configuration.
+To enable the B0 bootloader, select the ``SB_CONFIG_SECURE_BOOT_APPCORE`` Kconfig option in the sysbuild configuration.
+It automatically enables the ``SB_CONFIG_SECURE_BOOT_BUILD_S1_VARIANT_IMAGE`` option to construct the application binaries for both application's slots in non-volatile memory.
 
-The B0 bootloader additionally requires enabling the following options in the application configuration:
+The B0 bootloader additionally requires enabling the following options:
 
-* :kconfig:option:`CONFIG_SB_SIGNING_KEY_FILE` - Required for providing the signature used for image signing and verification.
-* :kconfig:option:`CONFIG_FW_INFO` - Required for the application versioning information.
-* :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` - Enable this option to set the version of the application after you enabled :kconfig:option:`CONFIG_FW_INFO`.
-  The nRF Desktop application with the B0 bootloader configuration builds two application images: one for the S0 slot and the other for the S1 slot.
-  To generate the DFU package, you need to update this configuration only in the main application image as the ``s1_image`` child image mirrors it.
-  You can do that by rebuilding the application from scratch or by changing the configuration of the main image through menuconfig.
-* :kconfig:option:`CONFIG_BUILD_S1_VARIANT` - Required for the build system to be able to construct the application binaries for both application's slots in non-volatile memory.
+* In the sysbuild configuration:
+
+  * ``SB_CONFIG_SECURE_BOOT_SIGNING_KEY_FILE`` - Required for providing the signature key file used for image signing and verification.
+    If this Kconfig option does not specify the signature key file, automatically generated signature key files will be used by default.
```

Comment:
```suggestion
    If this Kconfig option does not specify the signature key file, the application will use the default files.
```

Reply:
Discussed.
Decided to change whole bulletpoint to:
```
  * ``SB_CONFIG_SECURE_BOOT_SIGNING_KEY_FILE`` - Required for providing the signature key file used by the build system (to sign the application update images) and by the bootloader (to verify the application signature).
    If this Kconfig option does not specify the signature key file, debug signature key files will be used by default.
```